main/getting-started/alpha.md

```diff
@@ -0,0 +1,60 @@
+# Agoric Alpha	
+Release Version: Adriatic
+
+![Alpha Cover](./assets/alpha-cover.png)
+
+We’re building components to power the DeFi ecosystem. Come take a peek under the hood and help us scale our next-gen infrastructure. The Alpha Adriatic release provides you access to our toolset, API, and public chain.
+
+[Start Coding](https://agoric.com/documentation/getting-started/)
+[Join Community](https://discord.gg/gC9z6US)
+
+---------------------
+
+## What you can build in Alpha Adriatic 
+
+Numerous transaction components -- what we think of as DeFi Legos -- are available in the Alpha Adriatic release. We’ll be adding to the pool with our growing community’s help and participation in the coming months as we scale contributions. The initial slate of elements includes the following: 
+
+![Automated Market Maker](./assets/amm.png)
+### Automated Market Maker
+Build a smart contract that acts as an on-chain automated DEX with liquidity. 
+
+![Autoswap](./assets/autoswap.png)
+### Autoswap
+Tap into our Uniswap-like, constant-product AMM that executes upon finding a match in the current liquidity pool.
+
+![Covered Call](./assets/covered-call.png)
+### Covered Call
+Create an option contract that enables the holder to buy a specific, previously escrowed asset at a predetermined strike price (before expiration).
+
+![Simple Exchange](./assets/simple-exchange.png)
+### Simple Exchange
+A basic exchange with an order book for one asset, priced in a second asset.
+
+## Anatomy of a Smart Contract
+A call option is the right (with no obligation) to buy digital assets at a predetermined price, called the strike price. This call option is "covered," meaning that the owner of the digital assets must put the assets in escrow. Escrow guarantees that the assets can be transferred without relying on the owner of the digital assets to keep their promise. The call option has an expiration date, when the opportunity is cancelled. The owner of the digital assets cannot remove the assets from escrow before the expiration date. Any number of assets of different brands can be escrowed under different keywords. The proposal must have an exit record with the key "afterDeadline": 
```

Comment:
"owner of the digital assets" -> "digital assets owner"
"relying on the owner of the digital assets" -> "relying on the owner"
"The owner of the digital assets cannot remove the assets" -> "The digital assets owner cannot remove their assets"

"Any number of assets of different brands can be escrowed under different keywords. The proposal must have an exit record with the key "afterDeadline": "    These are using various names and concepts that we've defined and newbies won't know. Rewrite to be conceptual for understanding; "You can escrow different kinds of assets under different keys. The option must specify an expiration time." [although that last is a bit redundant with "The call option has an expirations date"]